spec/discovery-indexer/reader/purlxml_parser_strict_spec.rb

```diff
@@ -177,19 +179,37 @@
       pm = described_class.new('', Nokogiri::XML(public_xml_no_content))
       expect(pm.send(:parse_image_ids)).to be_nil
       expect(pm.send(:parse_file_ids)).to be_nil
+      expect(pm.send(:parse_sw_image_ids)).to be_nil
+      expect(pm.send(:parse_sw_file_ids)).to be_nil
     end
 
-    it 'returns nil when content metadata is present but no image or page ids are present' do
+    it 'returns nil when content metadata is present but no image, page, or thumb resource types are present' do
+      public_xml_no_content = "<publicObject id='druid:aa111aa1111'>#{@rights_metadata}#{@identity_metadata}#{@blank_content_metadata}</publicObject>"
+      pm = described_class.new('', Nokogiri::XML(public_xml_no_content))
+      expect(pm.send(:parse_image_ids)).to be_nil
+      expect(pm.send(:parse_sw_image_ids)).to be_nil
```

Comment:
Suggestion:  I would add tests to show that parse_sw_image_ids returns the right thing (with druid in it) and that image_ids do not have the druid in them.